src/SearchRow.js

```diff
@@ -0,0 +1,25 @@
+import React, {useState} from "react";
+
+function SearchRow (props){
+    const [selected, SetColour] = useState("")
```

Comment:
This will work great!  But it is generally accepted these variables follow a naming convention, `variable` and `setVariable`.

- There a couple improvements to be made here, first decide on what this state is, is it `selected`, or is it a `colour`?
- The the `SetColour` need to start with a lower case `S`, we should name variables using camel case. 😄

Reply:
also changed it, thank you for advice